### PR TITLE
Creating FrontEnd Workflow for running the tests

### PR DESCRIPTION
**Pull request by milensski**
I have created the GitHub Action workflow file for running the tests for the frontend

It is configured for all branches to run on pushes and pull requests, let me know with what settings to configure it if this is not suitable.

### CODE REVIEW

File: .github/workflows/action-test-frontend.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+name: Frontend Tests
+
+on:
```
Comment (julkascript) [inline .github/workflows/action-test-frontend.yml:3]: Sorry for not putting it as a requirement in the Jira, but we would ideally want the tests to be run only on PRs to develop, master.

File: frontend/package.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
     "react-router-dom": "^6.18.0"
   },
   "devDependencies": {
+    "@actions/core": "^1.10.1",
```
Comment (julkascript) [inline frontend/package.json:26]: I don't think this package is needed (unless I miss something)?

Comment (milensski) [reply frontend/package.json]: I have removed it now, 
it was for me to locally test the workflow but it is not needed as I see I can test directly on the branch in github

File: .github/workflows/action-test-frontend.yml (hunk 1 above)
Comment (milensski) [reply .github/workflows/action-test-frontend.yml:3]: it is now changed to run only on PR on develop and master

**Review by julkascript — APPROVED** [review]

**Pull request merged**